rambo/app.py

```diff
@@ -383,6 +383,7 @@ def up(ctx=None, **params):
     params['ram_size'], params['drive_size'] = options.size_option(
         params.get('ram_size'), params.get('drive_size')) # both ram and drive size
     params['sync_dir'] = options.sync_dir_option(params.get('sync_dir'))
+    params['address'] = options.address_option(params.get('address'))
```

Comment:
Could you put this at the top of the list so it's sorted? For a lot of stuff I wouldn't care, but I expect this list to get pretty long and that keep it tidier.

Some parts of cli.py should be sorted too, but that can wait for when the conf file loader thing is refactored. Doesn't make sense right now.